src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionRecordTypeParameterNameTest.java

```diff
@@ -41,7 +41,7 @@ protected String getCheckName() {
     @Test
     public void testOne() throws Exception {
         final File fileToProcess = new File(getNonCompilablePath(
-                "SuppressionXpathRegressionRecordTypeParameterName1.java"));
+                "InputXpathRecordTypeParameterNameExtend.java"));
```

Comment:
```suggestion
                "InputXpathRecordTypeParameterNameTypeDeclared.java"));
```

Reply:
I don't see the point why you would name it like that. Could you explain it a bit? Basically, why would you add "TypeXXXX" to the file name?

Reply:
The test is testing the generic var with `extends` and `&` synatx.

Reply:
Please read about TypeVariables in java.
https://docs.oracle.com/javase%2F7%2Fdocs%2Fapi%2F%2F/java/lang/reflect/TypeVariable.html

Make changes if you feel my suggestions make more sense than suffixes "Extend" and "Default"

Reply:
Sorry for the late reply, done